install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/templates/securedrop_rqrequeue.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=SecureDrop rqrequeue process
+After=redis-server.service
+Wants=redis-server.service
+
+[Service]
+Environment=PYTHONPATH="{{ securedrop_code }}:{{ securedrop_venv_site_packages }}"
+ExecStart={{ securedrop_venv_bin }}/python /var/www/securedrop/scripts/rqrequeue --interval 60
```

Comment:
a couple paths are hardcoded here in these services (better to use the variables where we can): 

`/var/www/securedrop` -> `{{ securedrop_code }}`
`/var/lib/securedrop` -> `{{ securedrop_data }}`

Reply:
Good catch. Fixed.